.github/workflows/cover.yaml

```diff
@@ -39,9 +39,10 @@ jobs:
         github-token: ${{ secrets.GITHUB_TOKEN }}
         flag-name: run-c-lcov
         parallel: true
-        github-branch: ${{ github.ref_name }}
+        git-branch: ${{ github.ref_name }}
 
     - name: Coveralls Erl
+      if: false
```

Comment:
I had an issue with submitting two cover data to coverall and letting it merge the cover data. 
so for now, upload the C part.

Reply:
thanks for advice! I tested in my fork which has no pull_request on the branch, trouble remains.
Iwill play with that coveralls API with curl when I have free time.

Reply:
solved! https://coveralls.io/builds/60634201